concepticondata/conceptlists.tsv

```diff
@@ -314,3 +314,4 @@ Yao-2017-1100	Yao, Zhao and Wu, Jia and Zhang, Yanyan and Wang, Zhenhong	2017		1
 Grierson-1928-168	Grierson, George Abraham	1928		168	basic, questionnaire	English	South Asian languages	https://dsal.uchicago.edu/books/lsi	Grierson1928		This list of 168 items for more than 380 language varieties edited by Grierson [Grierson 1928](:bib:Grierson1928) from the comparative vocabulary volume.	2-337	
 Hartmann-2013-162	Hartmann, Iren and Haspelmath, Martin and Taylor, Bradley	2013		162	questionnaire	English	Global	http://valpal.info	Hartmann2013		The online database ValPaL (Valency Patterns Leipzig) was created to study behaviour of verbs in different valency contexts. The concepts in this list are meant to be representative of the verbal lexicon and are reported as showing distinctive syntactic behaviour within and across languages.		
 Lundgren-2020-275	Lundgren, Olof	2020		275	areal	English	The Tupí–Guaraní languages Omagua and Kokama and their closest relative Tupinambá	https://lup.lub.lu.se/student-papers/search/publication/9007022	Lundgren2020		This 275-item list underlies the result of cognate detection in the Tupí–Guaraní languages Omagua and Kokama and their closest relative Tupinambá. The initial lexical data was drawn from the Tupí–Guaraní Comparative Lexical Dataset ([Chousou-Polydouri et al. 2019](:bib:ChousouPolydouri2019)). The cognate data were then used for the phonological reconstruction of their most recent common ancestor, Proto-Omagua–Kokama–Tupinambá.	89-97	
+Ruhlen-1998-36	Ruhlen, Merritt	1998		36	basic, areal	English	Yeniseian and Na-Dene	https://www.pnas.org/content/pnas/95/23/13994.full.pdf	Ruhlen1998		The basic concepts in this list serve to compare Proto-Yeniseian and Proto-Na-Dene cognates which are not widely shared by other language families. The author argues that this many cognates of basic vocabulary can only be explained by a common ancestor language.	13995-13996	
```

Comment:
Perhaps "this large amount of cognates" instead of "this many cognates"?

Reply:
yes, otherwise it should be "these many cognates"